.github/workflows/node.yaml

```
@@ -0,0 +1,50 @@
+name: Run flatpak-node-generator tests
+
+on:
+  push:
+    paths:
+      - node/**
+  pull_request:
+    paths:
+      - node/**
+
+defaults:
+  run:
+    working-directory: node
+
+jobs:
+  check:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.7'
+          - '3.8'
+          - '3.9'
+          - '3.10'
+        poetry-version:
+          - '1.1.13'
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Configure git
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: ${{ matrix.poetry-version }}
+      - name: Install OS dependencies
+        run: sudo apt-get update && sudo apt-get install -y flatpak-builder
+      - name: Install Flatpak dependencies
+        run: >
+          flatpak --user remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
+          && flatpak --user install -y flathub
+          org.freedesktop.{Platform,Sdk{,.Extension.node{14,16}}}//21.08
```

```suggestion
        run: |
          flatpak --user remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
          flatpak --user install -y flathub org.freedesktop.{Platform,Sdk{,.Extension.node{14,16}}}//21.08
```

Personal taste, find this more readable.

---

ngl I have no idea why I used `>` here